package.json

```diff
@@ -164,6 +166,10 @@
                     "type": "object",
                     "description": "%AWS.samcli.deploy.bucket.recentlyUsed%",
                     "default": []
+                },
+                "aws.sam.suppressCodeLenses": {
+                    "type": "boolean",
+                    "description": "%AWS.configuration.suppressCodeLenses%"
```

Comment:
why "suppress" (less common name) instead of "disable" or "enable"?

Reply:
Could go with disable. Wanted to lean towards a name implying that you're actively hiding these (so we can have no setting === default behavior)

Reply:
From a user's perspective, having some settings named "enable" and others "disable" is confusing.

```
Enable foo
Disable bar
Enable baz
...
```

> so we can have no setting === default behavior)

Consistent, positive logic at the interface seems more important than this internal aesthetic.